internal/chart/application_chart_test.go

```diff
@@ -298,7 +294,7 @@ func TestNewApplicationChart(t *testing.T) {
 			require.Nil(t, err)
 
 			expectedFilename := filepath.Join(chartDirectory, fmt.Sprintf("%s.yaml", tt.wantYamlsFilename))
-			actualFilename := filepath.Join(chartDirectory, fmt.Sprintf("%s-output.yaml", tt.wantYamlsFilename))
+			actualFilename := filepath.Join(chartDirectory, fmt.Sprintf("%s.output.yaml", tt.wantYamlsFilename))
```

Comment:
it's easier to work with all these files because they are sorted in a way that <file.yaml> and <file.output.yaml> are neighbors.